src/wp-includes/block-supports/elements.php

```diff
@@ -64,8 +68,17 @@ function wp_render_elements_support( $block_content, $block ) {
 		$content              = substr_replace( $block_content, ' class="' . $class_name . '"', $first_element_offset + strlen( $first_element ) - 1, 0 );
 	}
 
-	return $content . $style;
+	// Ideally styles should be loaded in the head, but blocks may be parsed
+	// after that, so loading in the footer for now.
+	// See https://core.trac.wordpress.org/ticket/53494.
+	add_action(
+		'wp_footer',
+		function () use ( $style ) {
+			echo $style;
+		}
+	);
```

Comment:
As noted here https://github.com/WordPress/wordpress-develop/pull/1808#discussion_r744147911, Core does not allow anynomous functions as action/filter callbacks. Same problem exists as in the other example that it's too late in the cycle to reconsider how to expose `$style` to a named function callback.

Suggestions for performance (`static`) and structure for multi-line comments:
```suggestion
	/*
	 * Ideally styles should be loaded in the head, but blocks may be parsed
	 * after that, so loading in the footer for now.
	 * See https://core.trac.wordpress.org/ticket/53494.
	 */
	add_action(
		'wp_footer',
		static function () use ( $style ) {
			echo $style;
		}
	);
```